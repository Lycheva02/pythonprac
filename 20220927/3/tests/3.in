3,2 
17,906
0,1
1,1
