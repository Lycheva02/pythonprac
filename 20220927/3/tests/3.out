2,5
906,923
